aggregator/src/main/scala/ai/chronon/aggregator/base/SimpleAggregators.scala

```diff
@@ -1,35 +1,14 @@
-/*
```

Comment:
Is this intentional?

Reply:
Not intentional. Updating merge conflicts. Will tag you when everything is ready.